vertica-kafka-scheduler/README.md

```diff
@@ -249,9 +248,9 @@ vkconfig microbatch --add \
 After you [create a scheduler](#create-a-scheduler), launch the scheduler to begin scheduling microbatches. To launch a scheduler, execute a `docker run` command that does the following:
 - Mounts a configuration file as a volume.
 - Specifies the scheduler image name.
-- Mounts the local `vkafka-log-config-debug.xml` file in the container's `/opt/vertica/packages/kafka/config` directory. This file configures log messages to help troubleshoot scheduler issues.
-- Mounts the local `/log` directory in the container's `/opt/vertica/log` directory to write logs to help troubleshoot scheduler issues.
-- Passes the Docker `--user` command to specify the user.
+- Mounts `vkafka-log-config-debug.xml` from the current directory file in the container's `/opt/vertica/packages/kafka/config` directory. This file configures log messages to help troubleshoot scheduler issues.
```

Comment:
"the current directory file" sound strange to me.  Should the word "file" be removed?
Should we use "in" or "into" as in the line below?  Or maybe the word "as" would work if we changed "mount" to "map".  Something like this might read better:
`maps <file> from the current directory as <file> in the container`

Reply:
yes, I think I should have left 'file' out--will remove.

So, for 'mount' --> 'map', I think it might be better to use 'mount' because the '-v' flag is the docker bind mount command, and it might make sense to keep the language 'official' or consistent? Maybe this is overthinking/incorrect, but I kinda see mapping as an association btwn two existing files, rather than copying an existing host file to a different location.

Either way, I revised the sentence--if you still think 'mapping' makes better sense after reading this comment and the revision, we can use 'mapping'.